python_files/supplementary_plot_helpers.py

```diff
@@ -406,3 +409,106 @@ def stitch_before_after_norm(
 
     pre_norm_tiled.save(pre_norm_stitched_path)
     post_norm_tiled.save(post_norm_stitched_path)
+
+
+def euclidean_timepoint(tp_one_data: pd.Series, tp_two_data: pd.Series) -> float:
+    """Compute the Euclidean distance between two timepoint data
+
+    Args:
+        tp_one_data (pd.Series):
+            The data for the first timepoint
+        tp_two_data (pd.Series):
+            The data for the second timepoint
+
+    Returns:
+        float:
+            The Euclidean distance between the two timepoint datapoints
+    """
+    return np.abs(tp_two_data - tp_one_data).mean()
```

Comment:
Isn't Euclidean distance $f(x,y) = \sqrt{(x-y)^2}$, or is this another other type of it?